packages/website/docs/examples/alert-block.md

```diff
@@ -0,0 +1,464 @@
+---
```

Comment:
Yeah I'm actually not sure what to do about this one. Normally we get the docs website theme from `getTheme` and just force light/dark mode on the editor. But here, part of the demo is that the alert block's styling changes automatically based on the system theme, and this does indeed work. Though to show this off, you can't force light/dark mode since you have to pass in both a light & dark theme for the auto switching to work.

So the tl;dr is we have 2 options:
- The theme changes automatically with the system theme, but ignores the docs website theme, which looks ugly.
- The theme is forced light/dark based on the docs website theme, but doesn't change with the system theme, so that feature isn't demoed.

I think the same issue happens in one of the theming demos too.

Reply:
I think I'd go for this :

> The theme is forced light/dark based on the docs website theme, but doesn't change with the system theme, so that feature isn't demoed.

Because nobody is going to change his system theme to try the demo. It's better to do this and explain how to use the toggle to change the example, and that if the theme is not passed explicitly, the system theme will be used